packages/java/tests/gradle/single-module/gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.daemon=false
-hillaVersion=24.6-SNAPSHOT
+hillaVersion=24.7-SNAPSHOT
```

Comment:
Is there a reason that flow shouldn't be updated to `24.7-SNAPSHOT`?

Reply:
No, but it is not related to this PR. It has some issues as you can see in #2961